docs/best_practices/integration_checklist.rst

```diff
@@ -0,0 +1,68 @@
+ToolShed Readiness Checklist
+============================
+
+The process from writing a tool to getting it on the ToolShed can be long and
+arduous and confusing. This checklist should assisst in making sure you have
+done everything required for a great, easy to use Galaxy Tool!
+
+Before ToolShed
+---------------
+
+- A GitHub repository should exist for your wrappers; either one you own, or
+  perhaps you are contributing to the IUC's repository.
+- A tool directory should exist for the specific set of tools or related
+  functionailty you are wrapping.
+- Check on BioConda for available packages which are required for the tool you
+  are wrapping. If they do not exist, you may need to create them. The IUC will
+  be happy to help you with doing this.
+- Planemo should be installed (``pip install -U planemo``)
+- You will need to have credentials to access your ToolShed (Either the `Main
+  ToolShed <https://toolshed.g2.bx.psu.edu/>`__, or your local Galactic ToolShed)
+
+Creating the Tool Wrapper
+-------------------------
+
+- Create your tool wrapper with a command like ``planemo tool_init --id 'tool_name' --name 'Tool description'``
+
+    - Find an appropriate name and ID by consulting the IUC's Best Practices List.
+
+- Fill in your requierments section in the Tool file with the conda package's name and version number.
+- Add test-data
+
+    - Fill test-data directory with at least one input file and the expected
+      output file. The primary output is a good choice for testing. Don't
+      forget the use of ``sim_size`` if variable data is included.
```

Comment:
Maybe add a suggestion on using small test data to save @martenson 's reviewing his nr. 1 review comment :)